src/pax.nim

```diff
@@ -103,25 +117,29 @@ if commonArgs.noColor.seen:
 
 # COMMANDS
 if spec.init.seen:
-  paxInit(force = initCmd.force.seen, skipManifest = initCmd.skipManifest.seen, skipGit = initCmd.skipGit.seen)
+  paxInit(force = initCmd.force.seen, skipManifest = initCmd.skipManifest.seen,
+      skipGit = initCmd.skipGit.seen)
 elif spec.list.seen:
   paxList(status = listCmd.status.seen, info = listCmd.info.seen)
 elif spec.add.seen:
-  paxAdd(input = addCmd.input.value, strategy = addCmd.strategy.value)
+  paxAdd(input = addCmd.input.value, noDepends = not addCmd.noDepends.seen,
```

Comment:
Oh, alright, that's why your `if` was inverted in the add command. Remove the ´not` here then, then the boolean variable accurately represents the value.